src/EventSubscriber/InitSubscriber.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Drupal\password_enhancements\EventSubscriber;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\Core\Datetime\DateFormatterInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Drupal\Core\Url;
+use Drupal\password_enhancements\PasswordChecker;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Subscribes to Drupal initialization event.
+ */
+final class InitSubscriber implements EventSubscriberInterface {
+
+  /**
+   * Account proxy.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $account;
+
+  /**
+   * Date formatter service.
+   *
+   * @var \Drupal\Core\Datetime\DateFormatterInterface
+   */
+  protected $dateFormatter;
+
+  /**
+   * Messenger service.
+   *
+   * @var \Drupal\Core\Messenger\MessengerInterface
+   */
+  protected $messenger;
+
+  /**
+   * Password checker service.
+   *
+   * @var \Drupal\password_enhancements\PasswordChecker
+   */
+  protected $passwordChecker;
+
+  /**
+   * Policy storage.
+   *
+   * @var \Drupal\password_enhancements\Entity\Storage\PolicyEntityStorageInterface
+   */
+  protected $policyStorage;
+
+  /**
+   * User storage.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface
+   */
+  protected $userStorage;
+
+  /**
+   * Initializes the init subscriber.
+   *
+   * @param \Drupal\Core\Session\AccountProxyInterface $account
+   *   The current user.
+   * @param \Drupal\Core\Datetime\DateFormatterInterface $date_formatter
+   *   Date formatter service.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   Entity type manager.
+   * @param \Drupal\Core\Messenger\MessengerInterface $messenger
+   *   Messenger service.
+   * @param \Drupal\password_enhancements\PasswordChecker $password_checker
+   *   Password checker service.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
```

Comment:
Yeah, but phpstorm always annoys me with these, it doesn't hurt, and in exchange phpstorm doesn't underline the whole comment block because that it is missing.

Reply:
Turn of this feature of PhpStorm instead ;P

Reply:
😒

Reply:
No follow-up task is created for this :)